src/golang/detectors/useless-if-condition/UselessIfCondition.go

```diff
@@ -2,7 +2,7 @@ package main
 
 import "fmt"
 
-// {fact rule=best-practices@v1.0 defects=1}
+// {fact rule=use-filepath-join@v1.0 defects=1}
```

Comment:
This seems incorrect. should be `useless-if-conditional`

Reply:
fixed